src/console_cmds.cpp

```diff
@@ -53,6 +53,24 @@
 /* scriptfile handling */
 static uint _script_current_depth; ///< Depth of scripts running (used to abort execution when #ConReturn is encountered).
 
+/* Scheduled execution handling */
+static std::string _scheduled_script; ///< Script scheduled to execute by the 'schedule' console command (empty if no script is scheduled)
```

Comment:
```suggestion
static std::string _scheduled_monthly_script; ///< Script scheduled to execute by the 'schedule' console command (empty if no script is scheduled)
```

Or something? I dunno .. but it might help to have it more clear. Or it might not :P

Reply:
Now changed